packages/server/src/api/controllers/static.js

```diff
@@ -62,6 +127,24 @@ exports.serveApp = async function(ctx) {
   await send(ctx, ctx.file || "index.html", { root: ctx.devPath || appPath })
 }
 
+exports.serveAttachment = async function(ctx) {
+  const appId = ctx.user.appId
+
+  const attachmentsPath = resolve(budibaseAppsDir(), appId, "attachments")
+
+  // Serve from CloudFront
+  if (process.env.CLOUD) {
+    const S3_URL = `https://cdn.app.budi.live/assets/${appId}/attachments/${ctx.file}`
+
+    const response = await fetch(S3_URL)
+    const body = await response.text()
+    ctx.body = body
+    return
```

Comment:
Maybe instead of returning here have the `attachmentsPath` and `send` in an else, that way making it clear those operations only occur when not running in the Cloud?